uw-frame-components/js/frame-config.js

```diff
@@ -348,7 +348,7 @@ define(['angular'], function(angular) {
             }
           },
           {
-            "name" : "uw-oskhosh",
+            "name" : "uw-oshkosh",
```

Comment:
This typo fix is a good catch; I think it deserves its own pull request (and its own JIRA issue to facilitate remembering to test this specifically as the software versions with this change migrate.)